browser-test/src/applicant/questions/map.test.ts

```diff
@@ -0,0 +1,163 @@
+import {test, expect} from '../../support/civiform_fixtures'
+import {
+  AdminPrograms,
+  AdminQuestions,
+  isLocalDevEnvironment,
+  loginAsAdmin,
+  logout,
+  validateAccessibility,
+  validateScreenshot,
+} from '../../support'
+import {Page} from 'playwright'
+
+const EXPECTED_LOCATION_COUNT = 5
+
+// map question tests rely on mock web services so they will only work in local dev environments
+if (isLocalDevEnvironment()) {
+  test.describe('map applicant flow', () => {
+    test.describe('single map question', () => {
+      const programName = 'Test program for single map'
+
+      test.beforeEach(async ({page, adminQuestions, adminPrograms}) => {
+        await setUpProgramWithMapQuestion(
+          page,
+          adminQuestions,
+          adminPrograms,
+          programName,
+        )
+      })
+
+      test('validate screenshot', async ({page, applicantQuestions}) => {
+        await test.step('Navigate to map question', async () => {
+          await applicantQuestions.applyProgram(programName, true)
+        })
+
+        await test.step('Take screenshot', async () => {
+          await validateScreenshot(page, 'map')
+        })
+
+        await test.step('Validate accessibility', async () => {
+          await validateAccessibility(page)
+        })
+      })
+
+      test('display map', async ({page, applicantQuestions}) => {
+        await test.step('Navigate to map question', async () => {
+          await applicantQuestions.applyProgram(programName, true)
+        })
+
+        await test.step('Wait for map to load and verify canvas', async () => {
+          const mapContainer = page.getByTestId('map-container')
+          await expect(mapContainer).toBeVisible()
+
+          const mapCanvas = mapContainer.getByRole('region', {name: 'Map'})
+          await expect(mapCanvas).toBeVisible({timeout: 10000})
```

Comment:
(blocking, question) Playwright will automatically wait for elements to become visible up to the timeout. There is almost no reason to need to manually override the default timeout (5000ms). Usually override it is indicative of some other issue. Can you tell me what you are running into?